static/src/stylesheets/pasteup/forms/_styles.scss

```diff
@@ -1,65 +1,7 @@
-@import '_forms.config';
-
-/*doc
----
-title: Forms
-name: forms
-category: Common
----
-
-```html_example
-<form class="form">
-    <fieldset class="fieldset">
-        <div class="fieldset__heading">
-            <h2 class="form__heading">
-                Form Heading
-            </h2>
-            <div class="form__note">
-                Form Note. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugiat, ratione!
-            </div>
-        </div>
-        <div class="fieldset__fields">
-            <ul>
-                <li class="form-field">
-                    <label class="label" for="field-1">Label</label>
-                    <input class="text-input" name="field-1" />
-                </li>
-                <li class="form-field">
-                    <label class="label" for="field-2">Another label</label>
-                    <input class="text-input" name="field-2" />
-                </li>
-                <li class="form-field">
-                    <div class="form-field__note">
-                        <a href="#">Forgotten your password?</a>
-                    </div>
-                    <button type="submit" class="submit-input">Submit</button>
-                    <label class="check-label check-label--helper">
-                        <input type="checkbox" />
-                        Remember me
-                    </label>
-                </li>
-            </ul>
-        </div>
-    </fieldset>
-</form>
-```
-*/
-
-/**
```

Comment:
Oh soz. I thought you meant the markup comment above. Stupid Github.

Yeah that probably is important.